frontend/src/components/SignupForm.tsx

```diff
@@ -65,6 +67,7 @@ const SignupForm = (props: any) => {
 
     if (isAuthenticated) {
       clearErrors();
+      clearModals();
```

Comment:
Again, why?

Reply:
explained in slack, can this conversation be resolved?

Reply:
Let's talk over the call. I would eventually be fine if there's a justification. I was just thinking ideally it would be nice to have 1 call for cancelling, but if not that's okay too.